about.html

```diff
@@ -0,0 +1,152 @@
+                  <h2 class="img-past-year">2020</h2>
+                  <p class="img-past-info">Making the Web Human-Centric</p>
+                </div>
+              </div>
+            </div>
+            
+            <div class="img-past2-container">
+              <div class="img-past2-overlay">
+                <div class="img-past-texts">
+                  <h2 class="img-past-year">2019</h2>
+                  <p class="img-past-info">Computer network and comunications</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        <!-- </div><img src="./Assets/websci2020.jpg"> -->
+        <!-- <img src="./Assets/websci2019.jpg"> -->
+        <div class="partners">
+          <h2 class="partners-title">Partners</h2>
+          <img src="Assets/icons8-line-32.png" alt="underline" class="underline">
+          <div class="all-sponsers">
+            <img src="./Assets/acm-logo.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner2.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/ica-logo.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner13.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner3.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner6.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner8.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner10.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner9.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/University_of_Southampton_Logo.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner7.png" alt="partner-logo" class="partner-logo">
+        <img src="./Assets/partner12.png" alt="partner-logo" class="partner-logo">
+            
+          </div>
+        </div>
+      </section>
+    <footer>
+      <section class="footer-mobile">
+        <div class="footer-container">
+          <div class="footer-logo-container">
+            <img src="./Assets/websci_white_logo.png" alt="logo" class="footer-logo">
+          </div>
+          <div class="footer-text">2021 ACM WEB SCIENCE. <br> Some rights deserved.</div>
+        </div>
+      </section>
+
+      <section class="footer-desktop-about">
+        <div class="footer-container">
+          <div class="footer-logo-container">
+            <a href="index.html"><img src="./Assets/websci_white_logo.png" alt="logo" class="footer-logo"></a>
+          </div>
+          <div class="footer-text">2021 ACM WEB SCIENCE.<br><br>Some rights deserved. All content on the website is licensed under a WebSci license. See <span><a href="https://webscience.org/privacy-policy/" class="privacy-link">https://webscience.org/privacy-policy/</a></span> for detailed privacy-policy conditions.</div>
+        </div>
```

Comment:
As stated by the previous reviewer, kindly wrap this section in a footer tag for semantic and HTML best practices. 👍